CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+**Features**:
+
+- Scrub span descriptions with encoded data images. ([#2560](https://github.com/getsentry/relay/pull/2560))
```

Comment:
I would omit this changelog records to be honest. And it's not really a feature in my opinion, maybe something like internal fits better here.

Reply:
Initially, I was planning to omit it. I saw the [changelog section in the README](https://github.com/getsentry/relay?tab=readme-ov-file#instructions-for-changelogs) and some scrubbing related entries in both features and internal. Since this is user-visible I'll leave it under features, unless someone strongly disagrees.